Automattic/wp-calypso
# Help Center: Move recommend resources to AI response location for forum

Fixes: https://github.com/Automattic/dotcom-forge/issues/8304

## Proposed Changes

* Move `Recommend resources` to AI generated  response page. The recommended resources will be placed above the AI response.



https://github.com/user-attachments/assets/2414cd8a-ddb3-453b-8f1b-c63f66de9b7f


## Why are these changes being made?


* Easier for users to see recommended resources when before asking for assistance on WP forums.

## Testing Instructions



* Checkout diff or use live link
* Log in with a free user
* Open help center, click `Still need help?`, enter a `subject` and text inside the `How can we help you?` text area, then click continue. The recommended resources should no longer appear below the forum contact form.
* Verify the recommended resources are updated and appear above the AI generated response (test with different values in `How can we help you?` text area)
* Test the back button navigation
* Regression test to ensure it didn't affect any other areas of the Help Center.

## Pre-merge Checklist
